internal/metric/otel_test.go

```diff
@@ -0,0 +1,25 @@
+// Copyright 2025 Democratized Data Foundation
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+//go:build telemetry
+
+package metric
```

Comment:
nitpick: Feel free to rename the package to telemetry at this point.

Reply:
done